docs/topics/about-ide-addons.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+
+[id="about-ide-addons_{context}"]
+= About the {ProductShortName} add-ons for IDEs
```

Comment:
Honestly, I don't think this info deserves its own module. I would add the 3 paragraphs to the IDE bullet point. Do you think it deserves its own module and subsection?